libs/exec/execv_windows.go

```diff
@@ -0,0 +1,95 @@
+//go:build windows
+
+package exec
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"sync"
+)
+
+// Note: Windows does not support an execv syscall that replaces the current process.
+// Thus we emulate that by launching a child process and streaming the output and returning
+// the exit code.
+// ref: https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/execv-wexecv?view=msvc-170
+func execv(opts ExecvOptions) error {
+	path, err := exec.LookPath(opts.Args[0])
+	if err != nil {
+		return fmt.Errorf("looking up %q failed: %w", opts.Args[0], err)
+	}
+
+	cmd := exec.Command(path, opts.Args[1:]...)
```

Comment:
what happens with stdin? Can you add a test that runs a script that reads stdin?

Reply:
Good catch! Fixed in windows.